### PR TITLE
test: allow to omit message in ok/is/isnt unit test helpers

### PR DESCRIPTION
**Pull request by locker**
The main goal of this PR is to make the message argument of ok/is/isnt unit test helpers optional because it's really annoying to specify it after each check.

Along the way, it also does some cleanup:
 - Removes duplicated code from `unit.h`.
 - Adds missing trailing semicolons after ok/is/isnt.

### CODE REVIEW

File: test/unit/unit.h [hunk 1]
```diff
@@ -90,41 +90,42 @@ _plan(int count, bool tap);
...
-	if (!res) {					\
-		_space(stderr);			\
-		fprintf(stderr, "#   Failed test '");	\
-		fprintf(stderr, fmt, ##args);		\
-		fprintf(stderr, "'\n");			\
-		_space(stderr);			\
-		fprintf(stderr, "#   in %s at line %d\n", __FILE__, __LINE__); \
-	}						\
-}
+#define _ok0(condition) _ok1(condition, "line %d", __LINE__)
```
Comment (alyapunov) [inline test/unit/unit.h:93]: As I said, I would call `_ok1(condition, "%s", #condition)`, but actually for ok case it does not matter.

**Review by alyapunov — APPROVED** [review]

File: test/unit/unit.h (hunk 1 above)
Comment (alyapunov) [reply test/unit/unit.h:93]: After a short discussion I was convinced that line number is always more informative.

**Review by alyapunov — APPROVED** [review]

**Review by nshy — APPROVED** [review]
LGTM

**Review by ochaplashkin — APPROVED** [review]
LGTM :rocket:

**Comment by locker** [comment]
Cherry-picked to 2.11 and 2.10.

**Pull request merged**